code/modules/mob/living/simple_animal/abnormality/zayin/fairy_festival.dm

```diff
@@ -99,9 +99,77 @@
 		user.gib()
 	return
 
+/mob/living/simple_animal/hostile/abnormality/fairy_festival/BreachEffect(mob/living/carbon/human/user, breach_type = BREACH_NORMAL)
+	if(breach_type == BREACH_PINK)
+		SummonGuys()
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/fairy_festival/proc/SummonGuys(again = TRUE)
+	var/mob/living/simple_animal/hostile/ordeal/pink_midnight/pink = locate() in GLOB.mob_living_list
+	for(var/i = 1 to 6)
+		var/turf/target_turf = get_turf(pink ? pink : src)
+		var/mob/living/simple_animal/hostile/mini_fairy/new_fairy = new(target_turf)
+		if(pink)
+			new_fairy.faction += "pink_midnight"
+	if(again)
```

Comment:
This feels obsolete, as you can just add a timer call in line 104.

Reply:
Yeah I uh... don't know why I did that. Fixing.